application/models/meeting.go

```diff
@@ -74,11 +75,89 @@ func (m *Meeting) ValidateUpdate(tx *pop.Connection) (*validate.Errors, error) {
 // FindByUUID finds a meeting by the UUID field and loads its CreatedBy field
 func (m *Meeting) FindByUUID(uuid string) error {
 	if uuid == "" {
-		return errors.New("error finding message: uuid must not be blank")
+		return errors.New("error finding meeting: uuid must not be blank")
 	}
 
 	if err := DB.Eager("CreatedBy").Where("uuid = ?", uuid).First(m); err != nil {
-		return fmt.Errorf("error finding message by uuid: %s", err.Error())
+		return fmt.Errorf("error finding meeting by uuid: %s", err.Error())
+	}
+
+	return nil
+}
+
+// FindCurrent finds the meetings that have StartDate before today an EndDate after today (inclusive on both)
+// Don't send testNow param for non-test code.
+// For testing with a different "now" time, use a one-entry slice, i.e. []time.Time{<my test time>}...
+func (m *Meetings) FindCurrent(testNow ...time.Time) error {
+	timeNow := time.Now()
+	if len(testNow) > 0 {
+		timeNow = testNow[0]
+	}
```

Comment:
I think I'd prefer `func (m *Meetings) FindOnDate(date time.Time) error` and called like `FindOnDate(time.Now())` so we don't have test-only code here. Or at least name it `FindOnDate` and change the comment to "If parameter list is empty, find date is Now".